packages/utils/src/getAccountsFromLocalStorage.ts

```diff
@@ -0,0 +1,22 @@
+import { AuthorizeResponse } from '@deriv/api-types';
+
+type TLocalStorageAccountsList = {
+    [k: string]: {
+        token: string;
+        accepted_bch: number;
+        landing_company_shortcode: string;
+        residence: string;
+        session_start: number;
+    } & NonNullable<NonNullable<NonNullable<AuthorizeResponse['authorize']>['account_list']>>[number];
```

Comment:
```suggestion
type TLocalStorageAccountsList = {
    [k: string]: TLocalStorageAccount & NonNullable<NonNullable<NonNullable<AuthorizeResponse['authorize']>['account_list']>>[number];
```
Best to split this type. We might need to use the declaration separately